pointerevents/pointerevent_attributes_hoverable_pointers-manual.html

```diff
@@ -113,6 +124,54 @@
                         }
                     });
                 });
+
+
+                // ===================== Synthetic input =====================
+
+                var target = document.querySelector('#square1');
+                var targetRect = target.getBoundingClientRect();
+                var xPosition1 = targetRect.left + 5;
+                var yPosition1 = targetRect.top + 5;
+
+                target = document.querySelector('#innerFrame').contentDocument.querySelector('#square2');
+                var frameRect = document.querySelector('#innerFrame').getBoundingClientRect();
+                targetRect = target.getBoundingClientRect();
+                var xPosition2 = frameRect.left + targetRect.left + 5;
+                var yPosition2 = frameRect.top + targetRect.top + 5;
+
+                actions = [
+                  {
+                    'source': 'mouse',
+                    'actions': [
+                      {'name': 'pointerMove', 'x': xPosition1, 'y': yPosition1},
+                      {'name': 'pointerDown', 'x': xPosition1, 'y': yPosition1},
+                      {'name': 'pointerUp'},
+                      {'name': 'pointerMove', 'x': xPosition2, 'y': yPosition2},
+                      {'name': 'pointerDown', 'x': xPosition2, 'y': yPosition2},
+                      {'name': 'pointerUp'},
+                      {'name': 'pointerMove', 'x': 0, 'y': 0},
+                    ]
+                  },
+                  // No support for pen in this implementation yet.
+                  //{
+                  //  'source': 'pen',
+                  //  'actions': [
+                  //    {'name': 'pointerMove', 'x': xPosition1, 'y': yPosition1},
+                  //    {'name': 'pointerDown', 'x': xPosition1, 'y': yPosition1},
+                  //    {'name': 'pointerUp'},
+                  //    {'name': 'pointerMove', 'x': xPosition2, 'y': yPosition2},
+                  //    {'name': 'pointerDown', 'x': xPosition2, 'y': yPosition2},
+                  //    {'name': 'pointerUp'},
+                  //    {'name': 'pointerMove', 'x': 0, 'y': 0},
+                  //  ]
+                  //}
+                ];
+
+                new driver().sendPointerActions(actions).then(function(result) {
+                  console.log('Input was successful.');
```

Comment:
Perhaps we should update the UI somehow to show whether automation is being used, and output any errors from the automation in the test output?